supervised/loglizer/loglizer/models/InvariantsMiner.py

```
@@ -0,0 +1,296 @@
+        from Console Logs for System Problem Detection. USENIX Annual Technical 
+        Conference (ATC), 2010.
+
+"""
+
+import numpy as np
+from itertools import combinations
+from ..utils import metrics
+
+class InvariantsMiner(object):
+
+    def __init__(self, percentage=0.98, epsilon=0.5, longest_invarant=None, scale_list=[1,2,3]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)